src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheckTest.java

```diff
@@ -120,6 +120,7 @@ public void testSeparated() throws Exception {
         final DefaultConfiguration checkConfig = createModuleConfig(ImportOrderCheck.class);
         checkConfig.addAttribute("groups", "java.awt, javax.swing, java.io, java.util");
         checkConfig.addAttribute("separated", "true");
+        checkConfig.addAttribute("separatedStaticGroups", "true");
```

Comment:
@pbludov , please explain this update.
"under" is default for `option`.
statics are going together with types (no seprations between them). So defining `separatedStaticGroups=true` should have no effect in such mode.

If yes, we need to mention this in xdoc. If no .... please explain.

Reply:
You are right. The property `separatedStaticGroups=true` is irrelevant here. I need to investigate here.

Reply:
>`separatedStaticGroups=true` should have no effect in such mode.

done.